docs/installation_guide.md

```diff
@@ -823,7 +823,7 @@ On the WireCloud instance:
 2. Edit `settings.py`:
     - Remove `wirecloud.oauth2provider` from `INSTALLED_APPS`
     - Add `social_django` to `INSTALLED_APPS`
-    - Add `wirecloud.fiware.social_auth_backend.FIWAREOAuth2` to `AUTHENTICATION_BACKENDS`. example:
+    - Add `wirecloud.fiware.social_auth_backend.FIWAREOAuth2` and `django.contrib.auth.backends.ModelBackend` to `AUTHENTICATION_BACKENDS`. example:
```

Comment:
Hmm, ok, we have to improve documentation in this regard, but this is not correct. Using the `'django.contrib.auth.backends.ModelBackend'` authorization backend is, in fact, not required. You only need it if you want to authenticate users using the local user database. See [django documentation](https://docs.djangoproject.com/en/1.11/topics/auth/customizing/#specifying-authentication-backends) for more details.

Reply:
So in that case the documentation should add this duality when it is needed to specify an authorization backend.